docs/getting-started/package-infrastructure.md

```diff
@@ -0,0 +1,878 @@
+---
+title: Package Infrastructure
+toc: true
+weight: 6
+indent: true
+---
+
+# Package Infrastructure
+
+In a [previous section] we learned that Crossplane can be configured with new
+composite resources (XRs) that are [composed] of other resources, allowing you
+to define and offer resources that group and abstract infrastructure primitives.
+We use two special Crossplane resources to define and configure new XRs and
+XRCs:
+
+- A `CompositeResourceDefinition` (XRD) _defines_ a new kind of composite
+  resource, including its schema. An XRD may optionally _offer_ a claim.
+- A `Composition` specifies which resources a composite resource will be
+  composed of, and how they should be configured. You can create multiple
+  `Composition` options for each composite resource.
+
+XRDs and Compositions may be packaged as a _configuration_, that may be
+installed to Crossplane. In the examples below we will create a configuration
```

Comment:
```suggestion
installed to Crossplane. In the examples below we will create a `Configuration`
```

Reply:
I went back and forth about using `Configuration` rather than "configuration". I ended decided not to since folks are mostly interacting with them via `kubectl crossplane` in this guide, so the fact that a `Configuration` CR is created is a bit of an implementation detail (in this context) IMO. I'd prefer to leave this like it is unless you feel strongly.